src/sysnet/times.74

```diff
@@ -425,16 +454,16 @@ NHOSTS==100.			   ;Maximum number of hosts we can query.
 
 ;;; TCP TIME servers.
 
-TCPTAB:	[ASCIZ /FAMILJEN/]
-	[ASCIZ /CP/]
-	[ASCIZ /PI/]
-	[ASCIZ /DX/]
-	[ASCIZ /DU/]
-	[ASCIZ /KN/]
-	[ASCIZ /MD/]
-	[ASCIZ /NX/]
-	[ASCIZ /AXPKLH/]
-	[ASCIZ /PCKLH/]
+TCPTAB:	[ASCIZ /ES/]
```

Comment:
Emulated ITS systems are still poor time servers, but it's not as bad here since TIMES doesn't set the local time (it doesn't, right?)

Reply:
Plus I don't get an answer from ES. I know UP doesn't have a TCP TIME server currently.

Reply:
ES will answer via chaosnet, currently not via TCP because my firewall is blocking whatever port is used. I can open it up.